agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -204,7 +204,7 @@ private static void start(Instrumentation instrumentation) {
         }
 
         // initialize StatsbeatModule
-        StatsbeatModule.get().start(telemetryClient, config.internal.statsbeat.intervalSeconds, config.internal.statsbeat.featureIntervalSeconds);
+        StatsbeatModule.get().start(telemetryClient, config.internal.statsbeat.intervalSeconds, config.internal.statsbeat.featureIntervalSeconds, config.preview.authentication.enabled);
```

Comment:
I think this should check `config.preview.authentication.enabled && config.preview.authentication.type != null`

@kryalama can you remind me why we need `enabled` instead of just checking if `type` is present?

Reply:
why check both? @kryalama?

Reply:
Either config.preview.authentication.enabled or config.preview.authentication.type != null  should be enough. If  config.preview.authentication.enabled is true then the agent expects a valid authentication type. We have the enabled flag to be consistent with other configuration types and to skip config validation if the enabled flag is false.